content/en/agent/faq/windows-agent-ddagent-user.md

```diff
@@ -0,0 +1,91 @@
+---
+title: ddagentuser with the Windows Agent
+kind: faq
+disable_toc: true
+---
+
+**Starting with release `6.11`, the Windows Agent main components (i.e. Core, and APM/Trace) run under the `ddagentuser` account, created at install time, instead of running on prior version under the `LOCAL_SYSTEM` account.**
```

Comment:
```suggestion
**Starting with release `6.11.0`, the Core and APM/Trace components of the Windows Agent run under the `ddagentuser` account, created at install time, instead of running on prior versions under the `LOCAL_SYSTEM` account. If enabled, the Live Process component still runs under `LOCAL_SYSTEM`. **
```